src/server/dflycmd.cc

```diff
@@ -392,6 +338,26 @@ void DflyCmd::Expire(CmdArgList args, ConnectionContext* cntx) {
   return rb->SendOk();
 }
 
+void DflyCmd::ReplicaOffset(CmdArgList args, ConnectionContext* cntx) {
+  RedisReplyBuilder* rb = static_cast<RedisReplyBuilder*>(cntx->reply_builder());
+  string_view sync_id_str = ArgS(args, 2);
+
+  VLOG(1) << "Got DFLY REPLICAOFFSET " << sync_id_str;
+  auto [sync_id, replica_ptr] = GetReplicaInfoOrReply(sync_id_str, rb);
+  if (!sync_id)
+    return;
+
+  string result;
+  unique_lock lk(replica_ptr->mu);
+  for (size_t flow_id = 0; flow_id < replica_ptr->flows.size(); ++flow_id) {
+    const auto& streamer = replica_ptr->flows[flow_id].streamer.get();
+    if (streamer) {
+      result = absl::StrCat(result, flow_id, streamer->GetRecordCount());
+    }
```

Comment:
I hope there are still spaces somewhere in the concatenated string? 😃

Reply:
By the way, absl::StrAppend is the better function here, but its not important in debug code

Reply:
There will be commas, I added them to the function but not pushed yet. Will be pushed soon with all the PR fixes